lib/tdf3/src/client/builders.ts

```diff
@@ -1,16 +1,16 @@
 import { S3Client, GetObjectCommand, HeadObjectCommand, S3ClientConfig } from '@aws-sdk/client-s3';
 import axios from 'axios';
-
+import { toWebReadableStream } from 'web-streams-node';
```

Comment:
Instead of doing this, just add a note telling users to first run their node stream through `streams.Readable.toWeb()`.
```suggestion
```

Reply:
The more built-ins we use the better! Thanks for catching this! I'll implement and test all your streams.readable.toWeb() feedback!